tools/tools.go

```diff
@@ -1,12 +0,0 @@
-//go:build tools
```

Comment:
I think this broke main, I cant run make lint here as the make file was never updated correctly